criu/include/pagemap.h

```diff
@@ -58,6 +58,9 @@ struct page_read {
 	/* Whether or not pages can be read in PIE code */
 	bool pieok;
 
+    /* Whether or not disable image deduplication*/
+    bool disable_dedup;
+
```

Comment:
@liusdu Would you be able to fix this patch to use tabs instead of spaces for indentation?

Reply:
ok, @rst0git sorry to this~